uk.co.compucorp.civicrm.hrcontactactionsmenu/templates/CRM/ContactActionsMenu/Page/Inline/ActionsPart2.tpl

```diff
@@ -0,0 +1,122 @@
+              <button type="button" class="btn btn-primary-outline">
+                <span class="btn-icon"><i class="fa fa-search"></i></span> View Entitlements
+              </button>
+            </p>
+            <hr>
+            <p><strong>Leave Approver(s):</strong> </p>
+            <p><a href="#" class="text-primary">John Snow</a></p>
+            <p><a href="#" class="text-primary">Allan Wite</a></p>
+            <button type="button" class="btn btn-secondary-outline">Manage Leave Approver</button>
+          </div>
+          <div class="col-md-4">
+            <h3>Workflows:</h3>
+            <p>
+              <button type="button" class="btn btn-primary-outline">
+                <span class="btn-icon"><i class="fa fa-user-plus"></i></span> Joining
+              </button>
+            </p>
+            <p>
+              <button type="button" class="btn btn-primary-outline">
+                <span class="btn-icon"><i class="fa fa-user-times"></i></span> Exiting
+              </button>
+            </p>
+            <p>
+              <button type="button" class="btn btn-primary-outline">Other...</button>
+            </p>
+            <hr>
+            <p>
+              <button type="button" class="btn btn-primary-outline">
+                <span class="btn-icon"><i class="fa fa-check-square-o"></i></span> New Task
+              </button>
+            </p>
+            <p>
+              <button type="button" class="btn btn-primary-outline">
+                <span class="btn-icon"><i class="fa fa-id-card-o"></i></span> New Document
+              </button>
+            </p>
+            <hr>
+            <p><strong>Line Manager(s):</strong> </p>
+            <p><a href="#" class="text-primary">John Snow</a></p>
+            <p><a href="#" class="text-primary">Allan Wite</a></p>
+            <button type="button" class="btn btn-secondary-outline">Manage Line Manager</button>
+          </div>
+          <div class="col-md-4">
+            <h3>Communicate </h3>
+            <p>
+              <button type="button" class="btn btn-primary-outline">
+                <span class="btn-icon"><i class="fa fa-envelope-o"></i></span> Send Email
+              </button>
+            </p>
+            <p>
+              <button type="button" class="btn btn-primary-outline">
+                <span class="btn-icon"><i class="fa fa-users"></i></span> Record Meeting
+              </button>
+            </p>
+            <p>
+              <button type="button" class="btn btn-primary-outline">
+                <span class="btn-icon"><i class="fa fa-file-pdf-o"></i></span> Create PDF Letter
+              </button>
+            </p>
+          </div>
+        </div>
+        <div class="row">
+          <hr>
+          <button type="button" class="btn btn-danger pull-right">
+            <span class="btn-icon"><i class="fa fa-trash"></i></span> Delete Staff Member
+          </button>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
There should be an empty line after every file. Please check whether your editor is using `editorconfig` properly